src/main/java/com/derongan/minecraft/mineinabyss/ascension/AscensionListener.kt

```diff
@@ -23,32 +23,25 @@ import java.util.*
 
 class AscensionListener : Listener {
     private val recentlyMovedPlayers: MutableSet<UUID> = HashSet()
-    private val worldManager: WorldManager? = Bukkit.getServicesManager().load(WorldManager::class.java)
-
-
-
+    private val worldManager: WorldManager = Bukkit.getServicesManager().load(WorldManager::class.java)!!
 
     @EventHandler(ignoreCancelled = true)
     fun onPlayerMove(moveEvent: PlayerMoveEvent) {
         val player = moveEvent.player
         val from = moveEvent.from
-        val to = moveEvent.to?:return
+        val to = moveEvent.to ?: return
 
         handleCurse(player, to, from)
     }
 
     @EventHandler(ignoreCancelled = true)
     fun onMove(moveEvent: VehicleMoveEvent) {
         val from = moveEvent.from
-        val to = moveEvent.to?:return
+        val to = moveEvent.to
 
-        for (passenger in moveEvent.vehicle.passengers){
-            if(passenger!=null && passenger is Player){
-                //compiler is dumb
-                val player = passenger as Player
-                handleCurse(player, to, from)
-            }
-        }
+        for (passenger in moveEvent.vehicle.passengers)
```

Comment:
Leaving out {s is not recommended. Java is not a whitespace dependent language in general so its kind of confusing when it becomes one.